joomla/joomla-cms
# [4.0] Namespace com_cpanel

Pull Request for Issue # .

### Summary of Changes
This simple PR convert cpanel component to namespace MVC in Joomla 4.

### Testing Instructions
1. Install Joomla 4.0-dev
2. Apply patch
3. Login to administrator area of the site, make sure cpanel is still loaded properly

### Additional comments
At the moment, our component dispatcher always check permission for backend component. However, today, I see that some component (like com_cpanel, com_login) doesn't need permission checking, so I move the code which check permission to a separate method to make it easier to for component to override if needed (in this case, no permission check needed, so the method can just be blank). Not sure if it is a right way, I am open for suggestion (just want to bring it for discussion to find the right way)
